### ISSUE TITLE
I.L. FE: Add Year screen to past limits flow

### ISSUE THREAD

**Issue by wesrowe** [department-of-veterans-affairs/va.gov-cms#13738]
### User story

**AS A** Veteran
**I WANT** to be asked what year I want income limits for if I clicked the past income limits CTA
**SO THAT** I can get the previous-year limits I want.

## Acceptance criteria

>Consider
> - Design / Accessibility reviews
> - Collab cycle requirements
> - Device sizes (mobile first)
> - Documentation updates / Change management
< - [Content model documentation](https://prod.cms.va.gov/admin/structure/cm_document)
> - Testing notes

- [ ] The Year screen is the first screen the user is taken to when they click the past income limits CTA
- [ ] Available years [DEFINE] are offered in the dropdown
- [ ] Continue button goes to Zip screen
- [ ] Back button returns to Start screen
- [ ] _**Should addition of Year to Review Info screen and Edit flow be their own tickets**_

**Comment by randimays** [comment]
Regarding the unit testing AC: as far as I can tell, it is not possible to target shadow DOM elements such as the `Continue` and `Back` buttons from a unit testing perspective. Tests for the edit functionality will be covered in the end-to-end tests.

**Comment by wesrowe** [comment]
Looks good. Only thing I couldn't test was how it sits in flow – because we don't have a start screen.

**Comment by wesrowe** [comment]
@thejordanwood, I noticed when you showed the IL sketch file that the Review info screen for previous year did _not_ include the ability to edit the year. I wrote this ticket to allow that, without realizing the discrepancy. Thoughts?

**Comment by thejordanwood** [comment]
@wesrowe Good catch. I don't have a reason for it not to be included. Should it be shown on both the current and past flows?

**Comment by wesrowe** [comment]
Good question @thejordanwood ... I hadn't thought of showing it for the current flow too. The user implicitly selected the year when they chose "current year." But I guess there's a chance it's confusing. We just asked them for 2022 zip and dependents, so will seeing "year = 2023" be confusing?

The current copy situation (I imagine) for current year with year editable – for me it implies that zip and dependents are from 2023:
- Year: 2023
- Zip: 11111
- Dependents: 3


What I'm imagining the clear (but bad, don't use it) copy could be:
- Income limit year: 2023 (edit)
- Zip code in 2022: 11111 (edit)
- Dependents in 2022: 3 (edit)

Do you want to mock that up for the previous year flow to start with?

Cc @cindymerrill

**Comment by wesrowe** [comment]
actually it's already mocked up in the app (although lorem ipsum)!
<img width="237" alt="image" src="https://github.com/department-of-veterans-affairs/va.gov-cms/assets/4054752/43f3fb59-422c-4c02-ab8e-6d1c99cd8271">

**Comment by cindymerrill** [comment]
I think it's confusing to show the year field in the Review screen for the present-year flow because the user didn't see a year-entry screen. I'd recommend leaving it off. @wesrowe

**Comment by randimays** [comment]
Just adding a note here that in staging right now, the "past" flow is hard coded in order to have completed the year screen ticket. This is because we don't have an intro screen to toggle the flow (current or past). The year will not show up on the review screen in the current flow, the way it is coded now.
